src/build-scripts/docker-centos.bash

```diff
@@ -0,0 +1,123 @@
+# docker run -ti --rm -v $(pwd):/jc quay.io/pypa/manylinux2014_x86_64:latest bash
+set -ex
+
+if [[ "$OSTYPE" != "msys" ]]; then
+    yum install ninja-build wget less ccache -y
+fi
+
+python -m pip install conan numpy --user
```

Comment:
`numpy` should be put as a build dependency in `pyproject.toml`. We'll actually need to use `oldest-supported-numpy`.

Reply:
What is the oldest supported numpy? I don't see it here https://docs.google.com/spreadsheets/d/1EwRlz5ZYObEOdBfIk8iTX5thlpTyEAfp3bxOgAfFOiU/edit#gid=1592527777

Reply:
I'm honestly not sure. Our numpy needs are so shallow that I'm not sure there really is a minimum. Whatever range works with pybind11, I guess?

Reply:
The oldest needed at compile time will be https://github.com/scipy/oldest-supported-numpy. We will want to use the `oldest-supported-numpy` package at build time for the reasons hilighted in https://github.com/scipy/oldest-supported-numpy.